### DISCUSSION TITLE
Prisma check if field is nullable

### DISCUSSION THREAD

**Discussion by rcMarty** [vsb-vaj/.github#11] [Q&A] [answered]
I can really easily get all fields in model via ``prisma.model.fields`` but is there any way to check which are optional and which are not?

**Comment by petrvecera** [comment, marked as answer]
If the project is in TypeScript, does it hint if it's mandatory or not? I am not sure. But in the plain JS I think there isn't a clear way to do it at the runtime.

**Reply by rcMarty** [reply]
Well it is in JS soo
